packages/components/src/drop-zone/provider.js

```diff
@@ -12,6 +12,8 @@ import isShallowEqual from '@wordpress/is-shallow-equal';
 const { Provider, Consumer } = createContext( {
 	addDropZone: () => {},
 	removeDropZone: () => {},
+	getDragData: () => {},
+	setDragData: () => {},
```

Comment:
For security reasons browsers don't allow the dropzones to access the "dragdata" content until the drop event is triggered. So this is a way to keep track of the data while dragging. Since we control both the draggable and the droppable, there's no issue in doing so.